assembl/static2/js/app/pages/brightMirrorFiction.jsx

```diff
@@ -104,6 +106,9 @@ export class BrightMirrorFiction extends Component<Props, State> {
     const modifyFictionCallback = (subject, body, state) => {
       this.setState({ title: subject, content: body, publicationState: state });
     };
+    const backBtnCallback = () => {
+      browserHistory.push(`${get('idea', { slug: slug, phase: phase, themeId: themeId })}`);
```

Comment:
Is there a reason why this solution was chosen over the incorporation of a Link component to this route inside of the BackButton component?

Reply:
Not really actually

Reply:
@MilanBarande Inside the Backbutton component, the Link would have displayed an anchor tag. I prefered to do a custom button (because there is 2 buttons, one for mobile, one for desktop)

Reply:
Allrighto